modules/apps/frontend-js/frontend-js-web/src/main/java/com/liferay/frontend/js/web/internal/js/importmaps/extender/FrontendJsWebJSImportMapsContributor.java

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
Would it make sense for this file to exist in `portal-language` modules?

Reply:
Mmm, I would say no because portal-language is a JAR with no code, AFAIK. It's only used as a container for lang keys. But it should be Brian who replies that, I guess...